public/components/notebooks/components/paragraph_components/para_input.tsx

```diff
@@ -133,40 +133,45 @@ export const ParaInput = (props: {
     return (
       <>
         <EuiFlexGroup alignItems="flexEnd" gutterSize="s">
-          <EuiFlexItem grow={6}>
-            <EuiCompressedFormRow label="Title" fullWidth>
-              <EuiCompressedComboBox
-                placeholder="Find visualization"
-                singleSelection={{ asPlainText: true }}
-                options={props.visOptions}
-                selectedOptions={props.selectedVisOption}
-                onChange={(newOption: EuiComboBoxOptionOption[]) => {
-                  if (newOption.length > 0) props.setVisType(newOption[0].className);
-                  props.setSelectedVisOption(newOption);
-                  props.setIsOutputStale(true);
-                }}
-              />
-            </EuiCompressedFormRow>
+          <EuiFlexItem grow={false} style={{ minWidth: '500px' }}>
+            <EuiFlexGroup gutterSize="s" alignItems="flexEnd">
+              <EuiFlexItem grow={true}>
+                <EuiCompressedFormRow label="Title" fullWidth>
+                  <EuiCompressedComboBox
+                    placeholder="Find visualization"
+                    singleSelection={{ asPlainText: true }}
+                    options={props.visOptions}
+                    selectedOptions={props.selectedVisOption}
+                    onChange={(newOption: EuiComboBoxOptionOption[]) => {
+                      if (newOption.length > 0) props.setVisType(newOption[0].className);
+                      props.setSelectedVisOption(newOption);
+                      props.setIsOutputStale(true);
+                    }}
+                  />
+                </EuiCompressedFormRow>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiSmallButton
+                  data-test-subj="para-input-visualization-browse-button"
+                  onClick={() => {
+                    setSelectableOptions([
+                      ...props.visOptions[0].options,
+                      ...props.visOptions[1].options,
+                    ]);
+                    setSelectableError(false);
+                    setIsModalOpen(true);
+                  }}
+                >
+                  Browse
+                </EuiSmallButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
           </EuiFlexItem>
+          <EuiFlexItem grow={true} />
```

Comment:
Not sure why is this needed?

Reply:
With: 
<img width="1070" alt="Screenshot 2024-08-28 at 2 20 05 PM" src="https://github.com/user-attachments/assets/c7a77738-e997-4688-bd00-f25320064de0">
Without: 
<img width="1180" alt="Screenshot 2024-08-28 at 2 21 41 PM" src="https://github.com/user-attachments/assets/96c6f4d0-80a9-4ae5-81d2-d39eaf359801">
Certain page sizes caused it to collapse to the left and not maintain the look of the rest of the page without the flex item.